run_data.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+"""
+CLI Functionality to generate reports and summaries of data sources.
+
+To generate model output, see ./run_model.py
+"""
+import pathlib
+import json
+import logging
+import click
+from libs import build_params
+from libs.datasets import JHUDataset
+from libs.datasets.dataset_utils import AggregationLevel
+
+WEB_DEPLOY_PATH = pathlib.Path("../covid-projections/public/data")
```

Comment:
would be cool if this could be set as opposed as part of the cli or via an env var

Reply:
yeah i think that's true, i can address that in a follow up pr if that's cool